docs/user/dashboard/lens-advanced.asciidoc

```diff
@@ -0,0 +1,259 @@
+[[create-a-dashboard-of-panels-with-ecommerce-data]]
+== Tutorial: Create a dashboard of panels with ecommerce sales data
+
+You collected sales data from your store, and you want to visualize and analyze the data on a dashboard. 
+To create dashboard panels of the data, open the *Lens* visualization builder, then  
+create the visualization panels that best display the data.
+
+When you've completed the tutorial, you'll have a dashboard that provides you with a complete overview of your ecommerce sales data.
+
+[role="screenshot"]
+image::images/lens_advanced_result.png[Dashboard view]
+
+[discrete]
+[[add-the-data-and-create-the-dashboard]]
+=== Add the data and create the dashboard
+
+To create visualizations of the data from your store, add the data set, then create the dashboard.
+
+. From the {kib} *Home* page, click *Try our sample data*.
+
+. From *Sample eCommerce orders*, click *Add data*.
+
+. Open the main menu, then click *Dashboard*.
+
+. On the *Dashboards* page, click *Create dashboard*.
+
+[float]
+[[open-and-set-up-lens]]
+=== Open and set up Lens
+
+Open the *Lens* editor, then make sure the correct fields appear.
+
+. From the dashboard, click *Create panel*.
+
+. On the *New visualization* window, click *Lens*.
++
+[role="screenshot"]
+image::images/lens_end_to_end_1_1.png[New visualization popover]
+
+. Make sure the *kibana_sample_data_ecommerce_* index appears.
++
+The list of fields that appear depend on the <<index-patterns,index pattern>>, <<set-time-filter,time filter>>, and field filters. 
+
+[discrete]
+[[view-the-number-of-transactions-per-day]]
+=== View the number of transactions per day
+
+To determine the number of orders made each day, create a bar chart, then add it to the dashboard.
+
+. Set the <<set-time-filter,time filter>> to *Last 30 days*.
+
+. From the *Available fields* list, drag and drop *Records* to the visualization builder.
++
+[role="screenshot"]
+image::images/lens_advanced_1_1.png[Added records to the workspace]
+
+. Change the *Vertical axis* title and display the number of orders per day. 
+
+.. From the editor, click *Count of Records*.
+
+.. In the *Display name* field, enter `Number of orders`.
+
+.. Click *Add advanced options > Normalize by unit*.
+
+.. From the *Normalize by unit* dropdown, select *per day*, then click *Close*.
```

Comment:
I think this is a bad choice of data to apply time scaling to, because we already set the time filter to Per 30 days. The best use for normalizing units is to scale time _down_ below what's normally possible, so maybe you could get the _approximate hourly value_ per day. Or maybe a per-minute value when looking at the last 24 hours. This is more useful for system metrics.

Reply:
When you have a time filter set to 'Last 30 days', the default interval is set to 12 hours so there's a change in the chart:

<img width="547" alt="Screenshot 2021-03-04 at 13 52 08" src="https://user-images.githubusercontent.com/4283304/109966683-d4da3e80-7cf0-11eb-805f-1d5c350f68b7.png">

<img width="561" alt="Screenshot 2021-03-04 at 13 51 56" src="https://user-images.githubusercontent.com/4283304/109966692-d73c9880-7cf0-11eb-8e78-0a4acc4bd9ff.png">

I would like to keep a time picker consistent for all the visualizations as we build a dashboard here and for example cumulative sum of products sold over weekend doesn't make sense for 24 hours. Not sure how to find the best solution for all.